x-pack/plugin/profiler/src/main/java/org/elasticsearch/xpack/profiler/ProfilingIndexManager.java

```diff
@@ -171,57 +264,117 @@ public void onResponse(CreateIndexResponse response) {
                         } else if (response.isShardsAcknowledged() == false) {
                             logger.warn("adding index [{}] for [{}], shards were not acknowledged", index, ClientHelper.PROFILING_ORIGIN);
```

Comment:
wrap arguments here for consistency

Reply:
Actually there's not much we can do: the formatting is enforced in the ES repo by Spotless (a Java formatter). Reformatting this manually would result in a violation (i.e. build error).